### PR TITLE
311 empty tm

### PR DESCRIPTION
**Pull request by VaabelMaret**
NOTE: BE task is on CR. Can't test right now.
https://github.com/keeleinstituut/tv-translation-order/pull/106/files

### CODE REVIEW

File: src/components/organisms/TranslationMemoriesSection/TranslationMemoriesSection.tsx
Comment (KaarelKa) [inline src/components/organisms/TranslationMemoriesSection/TranslationMemoriesSection.tsx:91]: **[blocker]**

Would be good to show a loader as well

File: src/hooks/requests/useTranslationMemories.ts
Comment (KaarelKa) [inline src/hooks/requests/useTranslationMemories.ts:400]: **[blocker]**

Can't merge right now, "undefined" will be added to the list of translationMemories, which will crash the app in TranslationMemorySection. This is waiting for BE to add the missing data to the response ?

**Review by KaarelKa — CHANGES_REQUESTED** [review]

**Comment by VaabelMaret** [comment]
ready to review, BE task is merged

**Pull request merged**
